src/components/RulesDialog.vue

```diff
@@ -1,240 +1,224 @@
+        </p>
 
-          <h4>Can I win by playing a King?</h4>
-          <p>
-            Yes. If you meet the required number of points, you win immediately. This means if you play a King
-            when you already have enough points to meet the new limit, you win on that turn.
-          </p>
+        <h4>Can I win by playing a King?</h4>
+        <p>
+          Yes. If you meet the required number of points, you win immediately. This means if you play a King
+          when you already have enough points to meet the new limit, you win on that turn.
+        </p>
 
-          <h4>If I play an Ace or a Six, are my cards destroyed as well?</h4>
-          <p>
-            Yes. Aces and Sixes destroy all Point Cards and all Face Cards plus Glasses Eights, respectively.
-            That includes any that you have out. Try to avoid destroying many of your own cards!
-          </p>
+        <h4>If I play an Ace or a Six, are my cards destroyed as well?</h4>
+        <p>
+          Yes. Aces and Sixes destroy all Point Cards and all Face Cards plus Glasses Eights, respectively.
+          That includes any that you have out. Try to avoid destroying many of your own cards!
+        </p>
 
-          <h4>Where can I play Cuttle?</h4>
-          <p>
-            You can play Cuttle anywhere you have a deck of cards and a friend! If you want to play online,
-            there is only one place:
-            <a href="https://www.cuttle.cards" target="_blank">www.cuttle.cards</a>
-          </p>
-        </section>
-      </v-card-text>
-      <v-card-actions class="d-flex justify-end">
-        <v-btn variant="outlined" color="primary" @click="close">Got It!</v-btn>
-      </v-card-actions>
-    </v-card>
-  </v-dialog>
+        <h4>Where can I play Cuttle?</h4>
+        <p>
+          You can play Cuttle anywhere you have a deck of cards and a friend! If you want to play online,
+          there is only one place:
+          <a href="https://www.cuttle.cards" target="_blank">www.cuttle.cards</a>
+        </p>
+      </section>
+    </template>
+    <template #actions>
+      <v-btn variant="outlined" color="surface-1" @click="close">Got It!</v-btn>
+    </template>
+  </base-dialog>
 </template>
 
 <script>
+import BaseDialog from '@/components/Global/BaseDialog.vue';
+
 export default {
   name: 'RulesDialog',
   emits: ['close'],
+  components: {
+    BaseDialog,
+  },
   props: {
     show: {
       type: Boolean,
       default: false,
     },
-    hideActivator: {
-      type: Boolean,
-      default: false,
-    },
   },
   data() {
     return {
       internalShow: false,
     };
   },
   computed: {
-    adaptedShow() {
-      return this.hideActivator ? this.show : this.internalShow;
-    },
     buttonSize() {
       return this.$vuetify.display.mdAndDown ? 'small' : 'medium';
     },
   },
   methods: {
     close() {
-      this.internalShow = false;
       this.$emit('close');
     },
+    open() {
+      this.$emit('open');
+    },
```

Comment:
I feel like I'm missing something, how does this get called? Is it something Vuetify is doing under the hood?

Reply:
I think it is something i forgot to remove, after removing it it still shows the appropriate behavior, thanks for the feedback !